homeassistant/components/deconz/gateway.py

```diff
@@ -30,6 +37,40 @@ def __init__(self, hass, config_entry):
         self.events = []
         self.listeners = []
 
+    @property
+    def bridgeid(self) -> str:
```

Comment:
We normally separate words with underscore: `bridge_id`.

Reply:
In this case I am following how the naming convention of the API